frontend/composables/useSortedMeetups.js

```diff
@@ -0,0 +1,55 @@
+export function useSortedMeetups (selectedMeetups, sortType) {
+  const oldMeetups = []
+  const futureMeetups = []
+  const today = new Date()
+  let sortedMeetups = []
+
+
+  function sort() {
+    for(const meetup of selectedMeetups) {
+      if(new Date(meetup.date) >= today ) {
+      futureMeetups.push(meetup)
+    }
+    else 
+      oldMeetups.push(meetup)
+    }
+  
+    function compare( a, b ) {
+      if ( a.date < b.date ) {
+        return -1
+      }
+      if ( a.date > b.date ) {
+        return 1
+      }
+      return 0
+    }
```

Comment:
There are still multiple codestyle issues here like missing curly braces or unneccessary spaces. I will create a issue to take care about it in next PR.

Reply:
I change it now, but eslint does not underline these issues